modules/reference/pages/systems/autoinst-distributions.adoc

```diff
@@ -8,15 +8,15 @@ The [guimenu]``Distributions`` page enables you to find and create custom instal
 The [guimenu]``Distributions`` page does not display distributions already provided.
 They can be found within the [guimenu]``Distribution`` drop-down menu of the [guimenu]``Autoinstallation Details`` page.
 
-Before creating a distribution, you must make an installation data available, as described in the [ref]_SUSE Linux Enterprise Deployment Guide_ link:https://www.suse.com/documentation/sles-12/singlehtml/book_sle_deployment/book_sle_deployment.html[{sle} Deployment Guide] or, respectively, link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/installation_guide/chap-kickstart-installations[{rhel} Installation Guide : chap-kickstart-installations]
+Installation data must be made available prior to creating a distribution, as described in the [ref]_SUSE Linux Enterprise Deployment Guide_ link:https://www.suse.com/documentation/sles-15/singlehtml/book_sle_deployment/book_sle_deployment.html#part.installserver[{sle} Deployment Guide] or, respectively, link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/installation_guide/chap-kickstart-installations[{rhel} Installation Guide : chap-kickstart-installations]
 This tree must be located in a local directory on the {productname} server.
 ====
 
 .Procedure: Creating a Distribution for Autoinstallation
 . To create a distribution, on the [guimenu]``Autoinstallable Distributions`` page click [guimenu]``Create Distribution`` in the upper right corner.
 . On the [guimenu]``Create Autoinstallable Distribution`` page, provide the following data:
 +
-** Enter a label (without spaces) in the [guimenu]``Distribution Label`` field, such as `my-orgs-sles-12-sp2` or ``my-orgs-rhel-as-7``.
+** Enter a label (without spaces) in the [guimenu]``Distribution Label`` field, such as `my-orgs-sles-15-sp1` or ``my-orgs-rhel-as-7``.
```

Comment:
And RHEL 8? ;)

Reply:
No, rhel8 is not yet supported

Reply:
thanks for clarifying it!  I'm not familiar with RH.

Reply:
@mcalmer Is there an ETA on that?

Reply:
No, we only know it will be difficult. So I expect it may take longer.